code/game/gamemodes/malfunction/newmalf_ability_trees/tree_networking.dm

```diff
@@ -92,32 +92,101 @@
 	if(!ability_prechecks(user, price))
 		return
 
-	var/title = input("Select message title: ")
-	var/text = input("Select message text: ")
-	if(!title || !text || !ability_pay(user, price))
-		user << "Hack Aborted"
-		return
+	var/reporttitle
+	var/reportbody
+	var/reporter = null
+	var/reporttype = input(usr, "Choose whether to use a template or custom report.", "Create Command Report") in list("Template", "Custom", "Cancel")
+	switch(reporttype)
+		if("Template")
```

Comment:
The stipulation was, let the AI only send messages from CCIAMS. The automated system. As to not have issues of staff complaints against CCIAA for something a malf did.